decidim-initiatives/spec/commands/decidim/initiatives/admin/update_initiative_type_scope_spec.rb

```diff
@@ -8,7 +8,7 @@ module Admin
       describe UpdateInitiativeTypeScope do
         let(:form_klass) { InitiativeTypeScopeForm }
 
-        context "when successfull update" do
+        context "when successful update" do
```

Comment:
the alternative would be `when successfully updated`, but I think this is better